python/mxnet/gluon/data/dataloader.py

```diff
@@ -52,7 +52,7 @@ class ConnectionWrapper(object):
     NDArray via shared memory."""
 
     def __init__(self, conn):
-        self.conn = conn
+        self._conn = conn
```

Comment:
?

Reply:
__getattr__ is dead cycle call. change the name same time.